server/web/sponsor/index.hbs

```diff
@@ -0,0 +1,37 @@
+<h1>Sponsor</h1>
+
+<p>
+  Development of jsPerf is generously supported by contributions from individuals and corporations. If you are benefiting from jsPerf and would like to help keep the project financially sustainable, please donate to a core contributor.
+</p>
```

Comment:
I agree randomly picking someone isn't great. I'm hesitant to point to an issue list because I'm afraid it'll change sponsorship into "bug/feature bounty." That's fine if that's what motivates someone to sponsor, but we also want people to keep in mind the time it takes to... 

1. respond to issues
2. review pull requests
3. update dependencies
4. monitor that the site is up
5. fight spam

That ongoing effort is what they're sponsoring.